### PR TITLE
Revise already edited chapters

### PR DESCRIPTION
**Pull request by smart-alek**
@ozan?

Wanted to clean up these first two sections a bit before moving on:
- adjust tone to be a bit more measured
- clean up a few sentences for clarity

### CODE REVIEW

File: book/analysis/introduction.md [hunk 1]
```diff
@@ -146,5 +143,5 @@ including the programming language, compiler, OS, machine, and time of day.
 
 We need a more abstract way to characterize these algorithms with respect to
 their execution times, one that is independent of the program or computer
-being used. This measure could be used to judge the algorithm alone and to
+being used. This method could then be used to analyze a single algorithm and
```
Comment (smart-alek) [inline book/analysis/introduction.md:146]: Callback to the "method of analysis" we mentioned before; using the same words triggers memory recall better.

At least I tell myself that. :octopus:

File: book/index.md [hunk 2]
```diff
@@ -39,5 +38,5 @@ to be important. For instance, we do not cover sorting algorithms,
 a topic that rarely arises in practice, at all; instead, we spend more time
 walking you through interesting applications of tree and graph traversal
 algorithms. For this reason, we recommend that you work through the examples
-slowly, with a REPL or text editor open, verifying that you absorb the content
-at the depth required to appreciate its timeless utility.
```
Comment (smart-alek) [inline book/index.md:43]: I always thought this was super dramatic, so forgive me for replacing it with something more down-to-earth. :earth_americas:

File: book/index.md [hunk 3]
```diff
@@ -24,13 +24,12 @@ have reached the limits of their capabilities.
...
-only have a better vocabulary for comparing alternative strategies, they
-are also more generally cognizant of the trade-offs we implicitly make
-every day. Those who see recursion everywhere see recursion everywhere...
-:)
+Those who study trees are more likely to identify tree shapes in their own
+data, unlocking superior solutions to their problems. Those who have big O
+analysis in their toolkits not only have a better vocabulary for comparing
+alternative strategies, they are also more generally cognizant of the
+trade-offs we implicitly make every day. Those who see recursion everywhere see
+recursion everywhere... :)
```
Comment (smart-alek) [inline book/index.md]: Forgot while I was working, but maybe it's finally time to remove the smiley... :sob:

File: book/analysis/introduction.md [hunk 4]
```diff
@@ -95,16 +95,15 @@ Sum is 500000500000, required  0.1646299 seconds
 Not only does the pattern still hold, but there appears to be a correlation
 between the _size_ of the input and the _time_ needed to process that input.
 Every time we increase `n` by some factor, the run time increases by the same
-factor. I certainly don't have time to wait around for the sum of the
-first 100,000,000,000,000,000 integers (220 days).
+factor. At this rate, it would take 220 days to calculate the sum of the first
```
Comment (smart-alek) [inline book/analysis/introduction.md:98]: Remove 1st person nonsense in favor of WE. :family:

Comment (ozan) [inline book/analysis/introduction.md:98]: We like this improvement.

File: book/analysis/introduction.md [hunk 5]
```diff
@@ -65,7 +65,7 @@ Sum is 50005000, required  0.0019162 seconds
 Sum is 50005000, required  0.0019360 seconds
 ```
 
-Seems like the running time is fairly consistent and takes an average of
+It seems like the running time is fairly consistent and takes an average of
```
Comment (ozan) [inline book/analysis/introduction.md:68]: Seems like this is better.

File: book/analysis/introduction.md [hunk 6]
```diff
@@ -123,18 +122,16 @@ Sum is 50000005000000, required 0.00000095 seconds
 Sum is 5000000050000000, required 0.00000119 seconds
 ```
 
-There are a couple things to notice about this output.
+There are a couple things to notice about this output:
```
Comment (ozan) [inline book/analysis/introduction.md]: There are a couple _of_ things? Is it an Americanism to omit the "of"?

File: book/index.md (hunk 3 above)
Comment (ozan) [inline book/index.md]: yes :smiley: :gun:

**Comment by ozan** [comment]
@aleksharma12 looks great! :sheep: when ready.

File: book/analysis/introduction.md (hunk 6 above)
Comment (smart-alek) [inline book/analysis/introduction.md]: No, [you're right](https://painintheenglish.com/case/267/), I just screwed up. :wrench:

**Pull request merged**